src/Features/Core/Portable/UseObjectInitializer/AbstractUseObjectInitializerDiagnosticAnalyzer.Analyzer.cs

```diff
@@ -0,0 +1,282 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.CodeAnalysis.CodeStyle;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.LanguageServices;
+using System.Linq;
+using System.Collections.Immutable;
+
+namespace Microsoft.CodeAnalysis.UseObjectInitializer
+{
+    internal partial class AbstractUseObjectInitializerDiagnosticAnalyzer<
```

Comment:
This was just moving the inner type into its own file.  Nothing really changed here.